src/tiled/tileanimationeditor.cpp

```diff
@@ -258,6 +261,13 @@ const QVector<Frame> &FrameListModel::frames() const
     return mFrames;
 }
 
+void FrameListModel::setDefaultFrameTime(int duration)
+{
+    DEFAULT_DURATION = duration;
+    for(Frame &frame : mFrames)
+        frame.duration = duration;
```

Comment:
In my previous review I commented that I don't like this side-effect of `setDefaultFrameTime`. Please don't change the animation and just set the new default frame time (also this setter can be static then).